core/store/migrations/0034_add_webauthn_table.go

```diff
@@ -0,0 +1,33 @@
+package migrations
+
+import (
+	"gorm.io/gorm"
+)
+
+const up34 = `
+	CREATE TABLE web_authns (
```

Comment:
Will it ever get reaped or does this table grow in size forever?

Reply:
It should grow 1:1 with number of keys registered. It's not a log.